### PR TITLE
fix cls for linux, add quit option

### PR DESCRIPTION
**Pull request by mrundle**

### CODE REVIEW

File: GameCode.py [hunk 1]
```diff
@@ -352,7 +359,11 @@ def start():
             print("A country's tension goes up if a war is lost.")
             print("A country's tension goes down if a war is won.")
             print("If a country's tension is above 50%, a civil war may begin.")
-            
+
+        elif command == "!quit":
+                print("Exiting.")
+                return
```
Comment (jeffm89) [inline GameCode.py:365]: Wouldn't this just print "Exiting." instead of actually quitting?

Comment (cece95) [reply GameCode.py:365]: Actually the whole game is inside a function so the return statement will exit the function and thus terminate the game

Comment (jeffm89) [reply GameCode.py:365]: Cheers, that makes sense now.

**Comment by brumafriend** [comment]
I've added this in.

**Pull request closed without merging**